Dockerfile

```diff
@@ -1,24 +1,46 @@
-FROM ruby:2.7.2
-RUN apt-get update -qq && apt-get upgrade -y
-RUN apt-get install -y build-essential nodejs && apt-get clean
-RUN gem install foreman
+ARG base_image=ruby:2.7.2
 
-ENV GOVUK_APP_NAME government-frontend
-ENV PORT 3090
-ENV RAILS_ENV development
+FROM $base_image AS builder
 
-ENV APP_HOME /app
-RUN mkdir $APP_HOME
+ENV RAILS_ENV=production
 
-WORKDIR $APP_HOME
-ADD Gemfile* $APP_HOME/
-ADD .ruby-version $APP_HOME/
-RUN bundle install
+RUN apt-get update -qy && \
+    apt-get upgrade -y && \
```

Comment:
We shouldn't really be running `apt-get upgrade` in Dockerfiles but we're having to do so as a workaround until we have our base images sorted out. Could you pop in the TODOs from (e.g.) https://github.com/alphagov/frontend/blob/main/Dockerfile which signpost this, please? Thanks!